utils.go

```diff
@@ -402,34 +402,25 @@ func sanitizeRootfs(id, rootfs string) string {
 	return rootfs
 }
 
-// getLinuxHeaderMounts returns a list of read-only mounts of the host's linux kernel headers.
-func getLinuxHeaderMounts() ([]specs.Mount, error) {
+// getLinuxHeaderMounts returns a list of read-only mounts of the host's linux
+// kernel headers.
+func getLinuxHeaderMounts(kernelHdrPath string) ([]specs.Mount, error) {
 
-	kernelRel, err := sysbox.GetKernelRelease()
-	if err != nil {
-		return nil, err
-	}
-
-	kernelHdr := "linux-headers-" + kernelRel
-
-	path := filepath.Join("/usr/src/", kernelHdr)
-	if _, err := os.Stat(path); err != nil {
-		return nil, err
-	}
-
-	mounts := []specs.Mount{}
+	var src = kernelHdrPath
 
-	// follow symlinks as some distros (e.g., Ubuntu) heavily symlink the linux
-	// header directory
-	mounts, err = createMountSpec(
-		path,
-		path,
+	// Follow symlinks as some distros (e.g., Ubuntu) heavily symlink the linux
+	// header directory.
+	mounts, err := createMountSpec(
+		src,
```

Comment:
We need a comment here on why we use `kernelHdrPath` as both the source and destination of the mount: it's because we will in addition create a softlink from the expected kernel header dir in the sys container to the `kernelHdrPath`.

Reply:
Done.